scripts/load_fides_child_examples.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
again, only executing this manually - but may be good to integrate with some of the nox sessions.